scripts/clear-silk-metadata.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+set -eou pipefail
+
+
+asset_group="$1"
+[ -z "${asset_group}" ] && echo "Missing asset_group def" && exit 1
+
+# if [ ! -f "${asset_group}.asset-group.json" ]; then
+#     echo "Missing ${asset_group}.asset-group.json"
+#     exit 1
+# fi
```

Comment:
please don't check in commented code

Reply:
will revisit why I left that there I guess I just copied it from Ryan and did not notice

Reply:
removed